lib/fizzy/parser.hpp

```diff
@@ -33,6 +33,18 @@ inline parser_result<uint8_t> parse_byte(const uint8_t* pos, const uint8_t* end)
 
     return {*pos, pos + 1};
 }
+template <typename T>
+
+inline parser_result<T> read_float_const(const uint8_t* pos, const uint8_t* end)
```

Comment:
Should this be better named `parse_float_const` ?

Reply:
`read_constant()`? `read_constant_bytes()`? But maybe keep the original name for now.